runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -658,8 +663,9 @@ private void checkClusterId(@Nonnull Layout layout) {
         // We haven't adopted a clusterId yet.
         if (clusterId == null) {
             clusterId = layout.getClusterId();
-            log.info("Connected to new cluster {}", clusterId == null ? "(legacy)" :
-                    UuidUtils.asBase64(clusterId));
+            if (clusterId != null) {
```

Comment:
Whats the reason for this change?

Reply:
if clusterId is null, the logs are filled with this log message.
This is because the management service invalidates the layout every second.
```
 2018-01-16 18:35:46,776 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:47,778 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:48,778 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:49,776 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:50,780 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:51,779 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:52,776 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:53,777 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:54,774 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:55,777 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:56,778 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:57,778 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
 2018-01-16 18:35:58,777 INFO  [ForkJoinPool.commonPool-worker-2] o.c.r.CorfuRuntime - Connected to new cluster (legacy)
```